src/backend/tests/fixtures/mocks.ts

```diff
@@ -0,0 +1,8 @@
+import mockery from "mockery";
+import * as nodemailerMock from "nodemailer-mock";
+mockery.enable({
+  warnOnUnregistered: false,
+});
+mockery.registerMock("nodemailer", nodemailerMock);
+
+export default nodemailerMock;
```

Comment:
🐧 I'd probably do
```suggestion
export {nodemailerMock}
```
here instead since this file is called `mocks.ts` (more generic)